kyuubi-util-scala/src/main/scala/org/apache/kyuubi/util/reflect/ReflectUtils.scala

```diff
@@ -64,6 +66,27 @@ object ReflectUtils {
     }
   }
 
+  /**
+   * Get a method with the given name and argument classes from the given class.
+   * @param clz the class to get the method from
+   * @param methodName the method name from declared field names
+   * @param argClasses the classes of the arguments
+   * @return an unbound method that can be invoked later
+   */
+  def getMethod(clz: Class[_], methodName: String, argClasses: Class[_]*): UnboundMethod = {
```

Comment:
this does not simplify much code, can we inline it?

Reply:
> this does not simplify much code, can we inline it?

Makes sense, changed.